udata_hydra/datalake_service.py

```diff
@@ -56,13 +58,23 @@ async def process_resource(url: str, dataset_id: str, resource_id: str, response
             resource_id, dataset_id
         )
     )
+
     tmp_file = None
+    mime_type = None
+    sha1 = None
+    filesize = None
+    error = None
+
     try:
         tmp_file = await download_resource(url, response)
 
         # Get file size
         filesize = os.path.getsize(tmp_file.name)
 
+        # Get checksum
+        with open(tmp_file.name, 'rb') as f:
+            sha1 = hashlib.sha1(f.read()).hexdigest()
```

Comment:
Maybe nitpicking but we could compute sha1 in blocks instead of loading all the file at once (eg https://gist.github.com/techtonik/df09baeacbebc52d234b#file-sha1-py-L20)

Reply:
Done in https://github.com/etalab/udata-hydra/pull/32/commits/db281ed91270a70d1ddf3d88a903e0d8c1bf612f